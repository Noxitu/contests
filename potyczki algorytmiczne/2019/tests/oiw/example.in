3 5 7
0 1 123
1 0 0
4 8 0
2 2 16
2 3 0
18 19 17
3 0 0