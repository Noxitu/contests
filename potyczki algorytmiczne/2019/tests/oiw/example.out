NIE
TAK
NIE
TAK
NIE
TAK
NIE